qa/rgw/ignore-pg-availability.yaml

```diff
@@ -1,9 +1,11 @@
 # https://tracker.ceph.com/issues/45802
 # https://tracker.ceph.com/issues/51282
 # https://tracker.ceph.com/issues/61168
+# https://tracker.ceph.com/issues/62504
 overrides:
   ceph:
     log-ignorelist:
     - \(PG_AVAILABILITY\)
     - \(PG_DEGRADED\)
     - \(POOL_APP_NOT_ENABLED\)
+    - not have an application enabled
```

Comment:
I think this is typically achieved by a generic `overall HEALTH_` override because any of other warnings that are being ignored above can "strike" the same way.

Reply:
hmm, i do see tons of `overall HEALTH_` suppressions under qa/suites/; but in recent examples of our `POOL_APP_NOT_ENABLED` failures, i don't see any matches for `overall HEALTH_` in the teuthology.log

i'm also hesitant to add any catch-alls here, because i want the rados team to know about any new/unexpected cluster warnings from the rgw suite as soon as they show up. am i misunderstanding how that `overall HEALTH_` thing works?

Reply:
My understanding (I could be wrong!) is that the same health check description string (in this case "X pool(s) do not have an application enabled") can be output in two different ways.  One, via `Monitor::log_health()`, would have the health check designator (in this case `POOL_APP_NOT_ENABLED`) appended.  The other is the periodic summary output coming from `Monitor::do_health_to_clog()`.  In that case the health check designator is appended only if
```
  if (g_conf()->mon_health_detail_to_clog &&
      summary != health_status_cache.summary &&
      level != HEALTH_OK) {
```
condition holds.  This goes for all health checks, so e.g. `PG_DEGRADED` that you are also ignoring could show up in the periodic summary output as "Degraded data redundancy: X/Y objects degraded (Z%)", without the designator that you are matching against.  `overall HEALTH_` in the ignorelist gets around that.

Reply:
> this in the teuthology.log: `cluster [WRN] overall HEALTH_WARN 1 pool(s) do not have an application enabled`

Yup, I believe this is that periodic summary output coming from `Monitor::do_health_to_clog()`.

Reply:
This PR looks good to me, but is the action item here that we should add a `overall HEALTH_` suppression in place of the one added here or in supplement to it?